sourcegraph/cody
# Remove in-chat onboarding buttons for new chats

Show the in-chat onboarding buttons only for first time users who haven't created a new chat.

| Before | After |
| - | - |
| <img width="819" alt="Screenshot 2023-07-31 at 4 01 19 pm" src="https://github.com/sourcegraph/cody/assets/153/2934a6b5-a0f1-445d-ac07-9ebb7269286d"> | <img width="819" alt="Screenshot 2023-07-31 at 4 01 38 pm" src="https://github.com/sourcegraph/cody/assets/153/3b799f27-1e11-4d94-b8ea-da279388ee02"> |

## Test plan

- Clear history and view chat. Chat should have welcome buttons
- Create a new chat. Chat should not have welcome buttons
- Navigate chats with history, verified they don't have the welcome buttons
